### PR TITLE
Availability and mode tests for shared filesystems.

### PR DESCRIPTION
**Pull request by RVerschoren**
These are some preliminary tests for the shared filesystems (/data, /user, /scratch).

The reliance on environment variables in _tests/cue/src/shared_fs_list.py_ is admittedly a bit clunky. However, it avoids specifying the complete path for each site separately, as that may differ for each site and user (primarily looking at VSC_SCRATCH).

### CODE REVIEW

File: tests/cue/shared_fs.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    tags = {"antwerp"}
+
+    @run_after('init')
+    def set_param(self):
+        path = os.path.join(self.fs['mount'], self.site)
+        self.descr += path
+        mode = self.fs.get('mode')
+        if not mode:
+            # default: check if directory has rwxr-xr-x permissions 
+            mode = '755'
```
Comment (smoors) [inline tests/cue/shared_fs.py:57]: should we also check the mode of $VSC_{HOME,DATA,SCRATCH}?
in Hydra, the mode is 700

File: tests/cue/shared_fs.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    tags = {"antwerp"}
+
+    @run_after('init')
+    def set_param(self):
+        self.descr += self.fs['mount'] + " " + self.fs['envar'] + " " + os.environ['USER']
+        path = os.environ[self.fs['envar']]
+        if not path:
+            # Order is important for indexing
+            sites = ['brussel', 'antwerpen', 'leuven', 'gent']
+            account_site = sites[int(os.environ['USER'][3])-1]
```
Comment (smoors) [inline tests/cue/shared_fs.py:86]:
```suggestion
            sites = {'1': "brussel", '2': "antwerpen", '3': "leuven", '4': "gent"}
            account_site = sites[os.environ["USER"][3]]
```

File: tests/cue/shared_fs.py (hunk 1 above)
Comment (RVerschoren) [reply tests/cue/shared_fs.py:57]: Checking $VSC_{HOME,DATA,SCRATCH} permissions is postponed until further discussion. 
This first iteration of the tests should check availability of the directories, which should work in this PR.

File: tests/cue/readme.md [hunk 3]
```diff
@@ -15,4 +15,6 @@ Each entry of the document is structured as follows:
     * `re`: custom regular expression to extract the version number from the command output, default is `r'(?:(\d+\.(?:\d+\.)*\d+))'`
     * `modname`: load the specified module before executing the test. Packages requiring a module load in certain sites and are installed at system level in others can be handled by greedly adding the required module in this field. 
 
-## env.py
\ No newline at end of file
+## env.py
+
+## shared_fs.py
```
Comment (Lewih) [inline tests/cue/readme.md:20]: Description in the readme ?

Comment (Lewih) [reply tests/cue/readme.md:20]: https://github.com/vscentrum/vsc-test-suite/pull/8/commits/9f50bfdc719cbb0028b6ad97039018b23095f3e2 solves the issues

**Review by Lewih — APPROVED** [review]

**Pull request merged**